infra/staging/main.tf

```diff
@@ -119,3 +124,44 @@ resource "google_sql_user" "user" {
   instance = google_sql_database_instance.strapi_db_instance.name
   project = data.google_project.project.project_id
 }
+
+##############
+## Frontend ##
+##############
+
+# Create a new storage bucket for frontend assets
+resource "google_storage_bucket" "call_home_frontend" {
+  project = data.google_project.project.project_id
+  name = var.frontend_bucket_name
+  location = var.location
+  force_destroy = true
+  uniform_bucket_level_access = true
+
+  website {
+    main_page_suffix = "index.html"
+    not_found_page = "index.html"
+  }
+}
+
+# Add proper permissions to bucket
+resource "google_storage_bucket_iam_member" "frontend_bucket_viewer" {
+  bucket = google_storage_bucket.call_home_frontend.name
+  role   = "roles/storage.objectViewer"
+  member = "allUsers"
+}
+
+resource "google_storage_bucket_iam_member" "frontend_bucket_admin" {
+  bucket = google_storage_bucket.call_home_frontend.name
+  role   = "roles/storage.admin"
+  member = "projectOwner:${data.google_project.project.project_id}"
```

Comment:
Why do we need this permission? I'd assume that project owners already have admin permissions over the GCS bucket.

Reply:
Removed.